resources/function_help/json/m_max

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "m_max",
+  "type": "function",
+  "description": "Returns the maximum m (measure) coordinate of a geometry.",
+  "arguments": [ {"arg":"geom","description":"a geometry"}],
```

Comment:
```suggestion
  "arguments": [ {"arg":"geometry","description":"a geometry"}],
```

Reply:
(also applies to others -- I realise there's inconsistencies here across different functions, but we should standardise on non-abbreviated forms)